myosuite/envs/env_variants.py

```diff
@@ -131,7 +131,7 @@ def register_env_variant(env_id:str, variants:dict, variant_id=None, silent=Fals
 
 # Example usage
 if __name__ == '__main__':
-    import myosuite
+    import robohive
```

Comment:
this will not work in myosuite... revert to `import myosuite`

Reply:
this will not work with myosuite.

Reply:
should we remove it completely then?

Reply:
Lets leave it. It will cause more problems. otherwise.